### PR TITLE
TMA-599: Users Brick: disabling users by mistake

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/283/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1161/ : FAILURE in 3m 48s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/310/ : FAILURE in 20m 51s

**Comment by vhtien** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/284/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1162/ : FAILURE in 4m 11s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/311/ : FAILURE in 17m 53s

**Comment by panjan** [comment]
ok to test

File: lib/gooddata/lcm/actions/synchronize_users.rb [hunk 1]
```diff
@@ -255,9 +255,11 @@ def call(params)
                                              do_not_touch_users_that_are_not_mentioned: do_not_touch_users_that_are_not_mentioned)
                       end
 
+                      params.gdc_logger.debug("Working client ids are: #{working_client_ids.join(', ')}")
+
                       unless do_not_touch_users_that_are_not_mentioned
                         domain_clients.each do |c|
-                          next if working_client_ids.include?(c.client_id)
+                          next if working_client_ids.include?(c.client_id.to_s)
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/synchronize_users.rb]: Calls 'c.client_id' 5 times (DuplicateMethodCall)

File: lib/gooddata/lcm/actions/synchronize_users.rb [hunk 2]
```diff
@@ -255,9 +255,11 @@ def call(params)
                                              do_not_touch_users_that_are_not_mentioned: do_not_touch_users_that_are_not_mentioned)
                       end
 
+                      params.gdc_logger.debug("Working client ids are: #{working_client_ids.join(', ')}")
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/synchronize_users.rb]: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/286/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/2/ : SUCCESS in 2m 25s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/313/ : SUCCESS in 1m 59s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/96/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/3/ : SUCCESS in 48s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/314/ : SUCCESS in 1m 50s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/936/ : FAILURE in 10s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/706/ : SUCCESS in 30m 05s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/98/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/5/ : SUCCESS in 3m 42s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/316/ : SUCCESS in 3m 22s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/938/ : FAILURE in 8s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/708/ : SUCCESS in 31m 00s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/99/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/6/ : SUCCESS in 43s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/317/ : SUCCESS in 39s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/939/ : FAILURE in 2s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/709/ : SUCCESS in 24m 24s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/100/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/8/ : SUCCESS in 2m 12s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/319/ : SUCCESS in 1m 54s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/943/ : SUCCESS in 44m 25s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/712/ : SUCCESS in 20m 45s

**Pull request merged**